frontend/test/__support__/e2e/cypress.js

```diff
@@ -332,3 +332,19 @@ export function generateUsers(count, groupIds) {
 export function enableSharingQuestion(id) {
   cy.request("POST", `/api/card/${id}/public_link`);
 }
+
+/**
+ * Open native (SQL) editor and alias it.
+ *
+ * @param {string} alias - The alias that can be used later in the test as `cy.get("@" + alias)`.
```

Comment:
:heart: the doc & annotation!

Reply:
Thank you.
I realized I could improve it a bit. Will do it in one of the next PRs.